test/rx.t.cpp

```diff
@@ -83,48 +83,13 @@ int main (int, char**)
   ut.ok (r9.match (start, end, text),        "e there are matches");
   ut.is (start.size (), (size_t) 6,          "e == 6 matches");
 
-#if defined(DARWIN) || defined(CYGWIN) || defined(FREEBSD) || defined(OPENBSD)
```

Comment:
Deleting a bunch of test cases doesn't fill me with confidence! It looks like, at least for these platforms, some of the allowed syntax changes. The details of the changes should certainly be documented and tested, and we can evaluate the impact on Taskwarrior from there.

In particular, if `\d` doesn't work to match a digit anymore, that might affect a lot of users.

Reply:
This issue seems to be solved, too. I just had to stick with `ECMAScript` grammar that supports all test cases.

Reply:
Awesome! In that case, is this change expected to have no user-visible impact?

Reply:
As far as I can tell from [Ecmascript at cppreference.com](https://en.cppreference.com/w/cpp/regex/ecmascript) there is no user-visible impact.